server/container_create.go

```diff
@@ -850,6 +850,29 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 		specgen.AddBindMount("/etc/hosts", "/etc/hosts", options)
 	}
 
+	// bind mount default user defined secrets
+	defaultMounts, err := secretMounts(mountLabel, defaultMountCfgFile)
+	if err != nil {
+		return nil, err
+	}
+	containerMounts := containerConfig.GetMounts()
+	for _, m := range defaultMounts {
+		// if m has already been mounted, do not mount secrets
+		if isAlreadyMounted(containerMounts, m.Destination) {
+			logrus.Warn("%q has already been mounted; cannot override mount", m.Destination)
+		} else {
+			specgen.AddBindMount(m.Source, m.Destination, nil)
```

Comment:
don't we need options? (3rd argument)

Reply:
`AddBindMount` adds a `rw` option if options is empty. Is there a specific option we want to add?

Reply:
yup, you should see wheater the container is RO rootfs and use RO in that case.

Reply:
Or we could have options specified in the config.  Not sure if we should default RHEL/Secrets to RO especially where we are copying.

Reply:
which one are we going with? options in config or just checking for RO rootfs?